ag/ag.proto

```diff
@@ -203,12 +197,19 @@ message OrgRequest {
     string orgName = 1;
 }
 
-// depending on situation frontend will have all enrolled users or only those that are not members of any group
-// set filterOutGroupMembers go get list of students not belonging to any group for given course
+// Depending on situation front-end will display all the enrolled users or only those that are not members of any group.
```

Comment:
What about this:
_EnrollmentRequest contains information about a course and a list of enrolled users. Currently, the list of enrolled users may be all users enrolled in the course, or alternatively those that are not members of any group. The ignoreGroupMembers field indicates that the list of enrolled users should not belong to any group for the given course._

Note `ignoreGroupMembers` instead of `filterOutGroupMembers`.

Reply:
But EnrollmentRequest does not contain any lists? It's a request sent to the server with course ID and a pair of options.

Renamed the field.

Reply:
Sorry, my misunderstanding. But the doc comment should still be more clear and not make case-specific references as the first order of business (as pointed out in another comment from yesterday below); maybe say something along the lines:

_EnrollmentRequest is a request for enrolled users of a given course, whose enrollment status match those provided in the request. To ignore group members that otherwise match the enrollment request, set ignoreGroupMembers to true._

Reply:
Yeah, this sounds reasonable. Changed.